src/main/scala/io/qbeast/spark/delta/writer/Compactor.scala

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2021 Qbeast Analytics, S.L.
+ */
```

Comment:
These copyright notices should be removed but I see that they are also present in files outside the scope of this PR. Maybe we should open a new issue & PR to clean the headers.

Reply:
They are in all files :laughing: . Scalafmt is set to enforce them.

Why they should be removed? Is it because of the Copyright?

Reply:
They are only in the files under `src/main` but files under `core/..` do not have them. Neither on the auxiliary files used to build the projects.

My concern is, why do we have a Copyright saying that open-source code belongs to a company?

Reply:
I started to think about this because I saw that the file in `core` on the PR did not have the copyright notice. There are different approaches with different implications, it is something to think for the future. 

ASF recommends including the License header on each file (I guess that for people to understand how the code is licensed in an easier way) but it is not required. 

I'd suggest to keep the Copyright notice as it is and add it on the `core/` files.